frontend/src/features/admin-form/settings/mutations.ts

```diff
@@ -95,7 +95,7 @@ export const useMutateFormSettings = () => {
         // Show toast on success.
         const isNowPublic = newData.status === FormStatus.Public
         const toastStatusMessage = isNowPublic
-          ? `Your form is now open.\n\nIf you expect a large number of responses,  [AutoArchive your mailbox](https://go.gov.sg/form-prevent-bounce) to avoid losing any of them.`
+          ? `Your form is now open.\n\nStore your secret key in a safe place. If you lose your secret key, all your responses will be lost permanently.`
```

Comment:
Apologies if the issue was not written clearly! What we want is actually for the toast to be the original (autoarchive mailbox) message if the form is in email mode, and the new secret key message if the form is in storage mode. 

You'll need to add a differentiation between the two cases. This change would replace the toast intended for email mode forms.

Reply:
ok no problem, i will resolve now